ui/components/app/beta-header/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+import Typography from '../../ui/typography/typography';
+import { TYPOGRAPHY, COLORS } from '../../../helpers/constants/design-system';
+import { BETA_BUGS_URL } from '../../../helpers/constants/beta';
+
+import { getShowBetaHeader } from '../../../selectors';
+import { hideBetaHeader } from '../../../store/actions';
+
+const BetaHomeHeader = () => {
+  const t = useI18nContext();
+  const showBetaHeader = useSelector(getShowBetaHeader);
+
+  if (!showBetaHeader) {
+    return null;
+  }
+
+  return (
+    <div className="beta-header">
```

Comment:
Could we use a Box for this, and specify padding, display and possibly width using props instead of css? I'm not sure if 100%/width is supported so that may need to remain as css. The main motivator here is just to give more power to our design system in the long run when improvements to the Box component are made.

Reply:
Updated!  Thank you @brad-decker !